### DISCUSSION TITLE
How to set HelmRelease/Spec/post-Renderers

### DISCUSSION THREAD

**Discussion by xwang2713** [fluxcd/flux2#1612] [General] [answered]
The following doesn't work
 postRenderers:
    - kustomize:
        patchesStrategicMerge:
          - <patch yaml file name>
https://fluxcd.io/docs/components/helm/api/#helm.toolkit.fluxcd.io/v2beta1.Kustomize says it should be 
[]Kubernetes pkg/apis/apiextensions/v1.JSON
What is the syntax for this JSON format?

**Comment by xwang2713** [comment, marked as answer]
Resolved.
Just yaml format like HelmRelease/spec/values

**Comment by norman-zon** [comment]
Is it possible to read postRenderer kustomization yaml from a file instead of putting it right into the CR?
